index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+    <title>Assignment 1</title>
+    <link rel="stylesheet" href="CSS/style.css">
+</head>
+
+<body>
+    <div class="navbar">
+        <ul>
+            <li><a href="">About Me</a></li>
+            <li>|</li>
+            <li><a href="">Portfolio</a></li>
+        </ul>
+    </div>
+    <br>
+    <H1>Symone Lewis</H1>
+
+    <!-- horizontal rule -->
+    <hr>
+    
+    <div class="columns">
+
+       <h3>I ENJOY LIFE AS A DEVELOPER</h3>
+    <h4> I'm Symone Lewis, a Developer based in NYC.
+        I have ten years of experience 
+        in the graphic design world, 
+        specializing in the creation of responsive websites.
+    </h4>
+
+    <img src="starter_code/images/blank-profile-picture-973460_960_720.png" 
```

Comment:
@symxing I would rename this image file to something simpler and shorter in order to avoid issues.